tests/rptest/tests/partition_force_reconfiguration_test.py

```diff
@@ -388,13 +389,20 @@ def __init__(self, test_context, *args, **kwargs):
                              num_brokers=5,
                              *args,
                              **kwargs)
-        self.default_timeout_sec = 60
+        self.default_timeout_sec = 120
         self.rpk = RpkTool(self.redpanda)
-        self.admin = Admin(self.redpanda)
+        self.admin = Admin(self.redpanda, retries_amount=20)
```

Comment:
With failure injector 504s occasionally appear when I make an admin request to a live node A that fails to reach a dead node B. Since we stop and start nodes here too I wonder if we need to used `retry_codes=[503, 504]` here as well. Maybe run a test 10-100 times to find out?